redbot/core/core_commands.py

```diff
@@ -739,18 +738,46 @@ class Core(commands.Cog, CoreLogic):
     @_set.command()
     @checks.guildowner()
     @commands.guild_only()
-    async def adminrole(self, ctx: commands.Context, *, role: discord.Role):
-        """Sets the admin role for this server"""
-        await ctx.bot.db.guild(ctx.guild).admin_role.set(role.id)
-        await ctx.send(_("The admin role for this guild has been set."))
+    async def addadminrole(self, ctx: commands.Context, *, role: discord.Role):
+        """ Adds an admin role for this guild """
+        async with ctx.bot.db.guild(ctx.guild).admin_role() as roles:
+            if role.id in roles:
+                return await ctx.send(_("This is already an admin role."))
+            roles.append(role.id)
+        await ctx.send(_("That role was added as an admin role for this guild."))
```

Comment:
This line and L758, "That role was added" doesn't sound very good, maybe a simple "Role added successfully."?

Reply:
I went with something different here, let me know if it sounds better to you.